### PR TITLE
Cut 0.3.0 release

### PR DESCRIPTION
**Pull request by SuperQ**
Includes many PRs:
* https://github.com/prometheus/mysqld_exporter/pull/10
* https://github.com/prometheus/mysqld_exporter/pull/12
* https://github.com/prometheus/mysqld_exporter/pull/14
* https://github.com/prometheus/mysqld_exporter/pull/15
* https://github.com/prometheus/mysqld_exporter/pull/17
* https://github.com/prometheus/mysqld_exporter/pull/19
* https://github.com/prometheus/mysqld_exporter/pull/20
* https://github.com/prometheus/mysqld_exporter/pull/21
* https://github.com/prometheus/mysqld_exporter/pull/22
* https://github.com/prometheus/mysqld_exporter/pull/25

### CODE REVIEW

**Comment by SuperQ** [comment]
@matthiasr @wrouesnel One more test release before 1.0

**Comment by matthiasr** [comment]
Let's stick with semantic versioning and call it 0.3.0. It's fine to jump from 0.3 or 0.4 to 1.0 when the time comes.

**Comment by juliusv** [comment]
:+1: for what @matthiasr said. Don't just jump over versions please :) Btw., we have nothing with a 1.x version in Prometheus-land yet, so I'd be careful about that - give this thing a while in use at various places to be sure that there will no further breaking changes for a while.

**Comment by wrouesnel** [comment]
Which reminds me I need to rebase #23 ... which I'll do now.

**Comment by wrouesnel** [comment]
So it looks like I was running an old release when testing the fix ups for userstats. In this version userstats is broken till the second commit of #23 is merged (bad pointer arithmetic bug - tested and fixed properly now).

**Comment by SuperQ** [comment]
@matthiasr, @juliusv Ok, changed release version to 0.3.0.

**Comment by SuperQ** [comment]
@wrouesnel Ok, I'll wait for 0.3.0 until https://github.com/prometheus/mysqld_exporter/pull/23 is merged and integrated here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+## 0.3.0 / 2015-08-31
+
+BREAKING CHANGES: Most metrics have been renamed with Prometheus Subsystem names
```
Comment (juliusv) [inline CHANGELOG.md]: s/renamed/prefixed/?
s/Subsystem/subsystem/

**Comment by juliusv** [comment]
:+1: otherwise

**Comment by matthiasr** [comment]
:+1: otherwise

**Pull request merged**